lemmy_db/src/comment_report.rs

```diff
@@ -0,0 +1,192 @@
+use diesel::{dsl::*, pg::Pg, result::Error, *};
+use serde::{Deserialize, Serialize};
+
+use crate::{limit_and_offset, MaybeOptional, schema::comment_report, comment::Comment, Reportable, naive_now};
+
+table! {
+    comment_report_view (id) {
+      id -> Int4,
+      creator_id -> Int4,
+      comment_id -> Int4,
+      comment_text -> Text,
```

Comment:
I agree, does make sense to copy that for the ninja edits, didn't think of that, but also the current comment might not be worth removing.

So it might be good to also include / join to get the **current comment text**, IE join to get it so you'd have `current_comment_text`. Might be useful to show them both if they're different in the UI (IE, maybe someone corrects themselves, and the new comment shouldn't be removed).

Reply:
it was already being joined, not sure why it wasn't grabbing it actually